src/components/footer/index.md.njk

```diff
@@ -13,6 +13,8 @@ The footer provides copyright, licensing and other information about your servic
 
 {{ example({group: "components", item: "footer", example: "default", id: "default-1", html: true, nunjucks: true, open: false, size: "m"}) }}
 
+If you use the page template, you'll also get the footer without having to add it, as it's included by default. However, if you want to customise the default footer, read the [page template guidance about customising components](/styles/page-template/#changing-template-content).
```

Comment:
I think a lot of people come straight into the "components" section and might not know where to go to get the Page Template info (my instinct was "patterns", not "styles").

I wonder if people will read the first part of this and gloss over the second part because of the "however", and thus not be able to find the page template guidance.

I might be overthinking.